src/winrt/activation/catalog.cpp

```diff
@@ -30,21 +30,29 @@ struct component
     wstring metadata_path;
     HMODULE handle = nullptr;
     activation_factory_type get_activation_factory;
-	ABI::Windows::Foundation::ThreadingType threading_model;
+    ABI::Windows::Foundation::ThreadingType threading_model;
 
-    bool LoadModule() 
+    HRESULT LoadModule()
     {
         if (handle == nullptr)
         {
             handle = LoadLibraryW(module_path.c_str());
+            if (handle == nullptr)
```

Comment:
Any reason not to use RETURN_IF_WIN32_ERROR macros?

Reply:
Can't depend on WIL